helm/ffc-sfd-permissions/templates/config-map.yaml

```diff
@@ -1 +0,0 @@
-{{- include "adp-helm-library.azure-config-service-map" . -}}
```

Comment:
This will cause the Infra release to fail as a config-map is used by the App Configuration Provider. Should be put back in. Contact me to discuss further if needed.